ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -356,24 +412,19 @@ static int
 gcp_object_list_single(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *directory,
   const char *prefix, char ***object_list, uint32_t *count)
 {
-    WT_UNUSED(file_system);
-    WT_UNUSED(session);
-    WT_UNUSED(directory);
-    WT_UNUSED(prefix);
-    WT_UNUSED(object_list);
-    WT_UNUSED(count);
-
-    return 0;
+    return gcp_object_list_helper(
+      file_system, session, directory, prefix, object_list, count, true);
 }
 
 static int
 gcp_object_list_free(
   WT_FILE_SYSTEM *file_system, WT_SESSION *session, char **object_list, uint32_t count)
 {
-    WT_UNUSED(file_system);
-    WT_UNUSED(session);
```

Comment:
Should keep `WT_UNUSED(session)` and `WT_UNUSED(file_system)`

Reply:
updated to have maybe_unused: bb12a4668327a39c079d9cc858096238335da33e